pontoon/base/urls.py

```diff
@@ -49,7 +49,7 @@
         name='pontoon.profile'),
 
     # API: Toogle user profile attribute
-    url(r'^api/v1/user/(?P<email>[\w.%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4})/$',
+    url(r'^api/v1/user/(?P<username>[\w-]+)/$',
```

Comment:
I think this won't match usernames of already existing usernames, because they are base64 encoded strings.